src/ts/UIFactory.ts

```diff
@@ -356,11 +356,19 @@ function adsUILayout() {
             cssClasses: ['text-right'],
           }),
         ],
-        cssClasses: ['controlbar-top ad-controlbar-top'],
+        cssClasses: ['ad-controlbar-top'],
+
+      }),
+      new Container(<ContainerConfig>{
```

Comment:
I think we can remove the type cast here


```suggestion
      new Container({
```

Reply:
Address in 895ed81d36d15421f116395c4ba261f0cec44621